stable/sealed-secrets/templates/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "sealed-secrets.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Remove `namespace: {{ .Release.Namespace }}` 
 on all resources. It's unnecessary.